Source/ablastr/fields/IntegratedGreenFunctionSolver.H

```diff
@@ -90,7 +129,8 @@ namespace ablastr::fields
     computePhiIGF (amrex::MultiFab const & rho,
                    amrex::MultiFab & phi,
                    std::array<amrex::Real, 3> const & cell_size,
-                   amrex::BoxArray const & ba);
+                   amrex::BoxArray const & ba,
+                   bool do_2d_slices);
```

Comment:
Would it make sense to use the same name (`is_igf_2d_slices`) as in the other functions - instead of `do_2d_slices`?

Reply:
Yes, sure!